### PR TITLE
experiments/simbricks/orchestration/nodeconfig : added ExpEnv variable to AppConfig.config_files method

### PR DESCRIPTION
**Pull request by q713**

### CODE REVIEW

File: experiments/pyexps/farmem.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import simbricks.orchestration.experiments as exp
 import simbricks.orchestration.nodeconfig as nodec
 import simbricks.orchestration.simulators as sim
+from simbricks.orchestration.experiment.experiment_environment import ExpEnv
```
Comment (jonas-kaufmann) [inline experiments/pyexps/farmem.py:26]: Please never just import one class or function from a module. Instead always import the whole module. We use Google's Python style guide. See [here](https://google.github.io/styleguide/pyguide.html#22-imports) for more info on this.

I know that we currently still violate this in multiple places but I hope to resolve this some time soon.

File: experiments/pyexps/memswitch.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 import simbricks.orchestration.experiments as exp
 import simbricks.orchestration.nodeconfig as node
 import simbricks.orchestration.simulators as sim
+from simbricks.orchestration.experiment.experiment_environment import ExpEnv
```
Comment (jonas-kaufmann) [inline experiments/pyexps/memswitch.py:26]: Same here.

File: experiments/pyexps/memswitch_large.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
 import simbricks.orchestration.experiments as exp
 import simbricks.orchestration.nodeconfig as node
 import simbricks.orchestration.simulators as sim
+from simbricks.orchestration.experiment.experiment_environment import ExpEnv
```
Comment (jonas-kaufmann) [inline experiments/pyexps/memswitch_large.py:26]: Same here.

File: experiments/pyexps/sysbench_memory.py [hunk 4]
```diff
@@ -28,6 +28,7 @@
 import simbricks.orchestration.experiments as exp
 import simbricks.orchestration.nodeconfig as node
 import simbricks.orchestration.simulators as sim
+from simbricks.orchestration.experiment.experiment_environment import ExpEnv
```
Comment (jonas-kaufmann) [inline experiments/pyexps/sysbench_memory.py:31]: Same here.

File: experiments/pyexps/farmem.py (hunk 1 above)
Comment (jonas-kaufmann) [reply experiments/pyexps/farmem.py:26]: I also just noticed this with #132. Maybe you can add a commit here to fix it for #132 as well?

**Review by jonas-kaufmann — CHANGES_REQUESTED** [review]

**Review by marvin71 — APPROVED** [review]
Looks good

File: experiments/simbricks/orchestration/nodeconfig.py [hunk 5]
```diff
@@ -26,7 +26,7 @@
 import tarfile
 import typing as tp
 
-from simbricks.orchestration.experiment.experiment_environment import ExpEnv
+import simbricks.orchestration.experiment.experiment_environment as env
```
Comment (jonas-kaufmann) [inline experiments/simbricks/orchestration/nodeconfig.py:29]:
```suggestion
from simbricks.orchestration.experiment import experiment_environment as exp_env
```
This was unfortunately still the wrong form. Have a look at this section here: https://google.github.io/styleguide/pyguide.html#224-decision.

Also, I would have kept the variable name as `env` and then rename the imported module to `exp_env` but choose whatever you find better here.

**Review by jonas-kaufmann — CHANGES_REQUESTED** [review]

**Review by jonas-kaufmann — APPROVED** [review]

**Pull request merged**